src/main/java/fun/epak/pak/model/Post.java

```diff
@@ -21,6 +21,7 @@ public class Post {
     @ManyToOne
     private User user;
     private LocalDate createDate;
+    @Column(columnDefinition = "TEXT")
```

Comment:
A czy `@Lob` nie zadziała?
https://stackoverflow.com/questions/15135905/columndefinition-text-for-all-types-of-databases